doc/api_transaction.rst

```diff
@@ -1,5 +1,5 @@
 ..
-  Copyright (C) 2014-2016 Red Hat, Inc.
+  Copyright (C) 2014-2018 Red Hat, Inc.
```

Comment:
don't do this to files you didn't modify in 2018

Reply:
I am fine with the change. We overlooked it in past.